scheduler/clusterprocessor.go

```diff
@@ -78,10 +76,22 @@ func ClusterProcessor(commandFlag configuration.CommandFlag,
 
 		create := func() {
 			for _, cluster := range provider.Clusters {
-				err := runGCPCreate(cmc, ctx, cluster.Cluster)
+				clusterEndpoint, clusterCertCa, err := runGCPCreate(cmc, ctx, cluster.Cluster)
 				if err != nil {
 					color.Red(err.Error())
 				}
+
+				cluster.Cluster.Endpoint = clusterEndpoint
+				cluster.Cluster.CertCa = clusterCertCa
```

Comment:
as far as I understand, Endpoint and CertCa should NEVER be set by the config file. So I think we could fail if this variables have a value already:
```
if cluster.Cluster.Endpoint != "" {
    log.Error("Cluster Endpoint should not be set ==> ignoring its value")
}
```

Reply:
It is not set by the config file. I have update the file `platform/provider/gcp/create.go` due to return the endpoint and the certificate after the create of the cluster.
This two things have to be provide due to be able to login to the kubernete cluster.